quarkus.jdt.ext/com.redhat.microprofile.jdt.quarkus.test/projects/maven/scheduler-quickstart/src/main/java/org/acme/scheduler/CounterBean.java

```diff
@@ -27,7 +27,7 @@ void cronJob(ScheduledExecution execution) {
         System.out.println(execution.getScheduledFireTime());
     }
 
-    @Scheduled(cron = "{cron.expr}")
+    @Scheduled(cron = "{cron.expr}", every = "{cron.expr}")
```

Comment:
The CI build fails, I think you should write

`every = "{every.expr}` to avoid properties duplicate.